modules/xphoto/samples/grayworld_color_balance.cpp

```diff
@@ -0,0 +1,59 @@
+#include "opencv2/xphoto.hpp"
+
+#include "opencv2/imgproc.hpp"
+#include "opencv2/highgui.hpp"
+
+#include "opencv2/core/utility.hpp"
+
+const char* keys =
+{
+    "{i || input image name}"
+    "{o || output image name}"
+};
+
```

Comment:
please, put "using namespace cv; using namespace std;" and then remove cv:: and std:: to make the code cleaner